packages/admin-panel/src/VizBuilderApp/components/PreviewOptions.jsx

```diff
@@ -140,6 +140,13 @@ const UploadDataModal = ({ isOpen, onSubmit, onClose }) => (
   />
 );
 
+/**
+ * @returns ISO date string in the format "yyyy-mm-dd", or `null` if the input is not a valid `Date`
+ * instance.
+ * @param date A `Date` object.
+ */
+const getIsoDateString = date => (isNaN(date.getTime()) ? null : date.toISOString().slice(0, 10));
```

Comment:
Technically this doesn't return `null` if it's an invalid date, as if `date` is `null` or `undefined` then it will probably throw an error trying to run `getTime()` - possibly we could handle this as well, on the very slim chance it could happen?

Reply:
Great catch 🙏